plugins/env/env.go

```diff
@@ -82,8 +103,21 @@ type pluginProvider struct {
 	plugins.PluginProviderBase
 }
 
-func (pluginProvider) NewPlugin(plugins.PluginOptions) (plugins.Plugin, error) {
-	return plugin{}, nil
+func (pluginProvider) NewPlugin(options plugins.PluginOptions) (plugins.Plugin, error) {
+	var c config
+	if err := schematypes.MustMap(configSchema, options.Config, &c); err != nil {
+		options.Log.WithError(err).Error("Invalid configuration")
```

Comment:
Again no need to log on contract violations, caller is responsible for logging errors from schema mismatch